blocks/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+# Create your models here.
+
+
+class Block(models.Model):
```

Comment:
add class level description for each model

Reply:
done